graphql_api/types/coverage_analytics/coverage_analytics.graphql

```diff
@@ -13,13 +13,49 @@ type CoverageAnalytics {
   "PercentCovered is percent of lines covered (e.g., 87.25)"
   percentCovered: Float # formerly repository.coverage
 
-  "measurements are points in the time series for coverage over time"
+  "Measurements are points in the time series for coverage over time"
   measurements(
     interval: MeasurementInterval!
     after: DateTime
     before: DateTime
     branch: String
   ): [Measurement!]! # formerly repository.measurements
+
+  ## Flags ##
+  "Flags are the information about the flags for this repository"
```

Comment:
nit: idk how helpful this comment is tbh.

Objects is the information about the objects for this repository

similar story with the component one

Reply:
fixed!